src/ucs/debug/debug.c

```diff
@@ -899,8 +899,8 @@ static void ucs_error_freeze(const char *message)
     int ret;
 
     /* restore original SIGINT handler to allow termitate process via Ctrl+C */
-    sigemptyset(&sigact.sa_mask);
-    sigaction(SIGINT, &sigact, NULL);
+    sigemptyset(&orig_sigaction.sa_mask);
+    sigaction(SIGINT, &orig_sigaction, NULL);
```

Comment:
sorry fo confusion : i mean to use orig_sigaction function instead of "sigaction" (not change struct name)
pls grep for it in debug.c

Reply:
> sorry fo confusion : i mean to use orig_sigaction function instead of "sigaction" (not change struct name)
> pls grep for it in debug.c

ok, done... but I tested original PR where sigaction was called - it worked